src/category_theory/sums/default.lean

```diff
@@ -0,0 +1,168 @@
+/-
```

Comment:
Hmm... we usually don't put lots of theory in `default.lean`. Can you move this to `basic.lean`, and import that in `default.lean`? That seems to be the "correct" way of doing things...

Reply:
Sure.